src/module.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015 Foursquare Labs, Inc.
```

Comment:
The original says 2016. I think it should be the same here, but please correct me if I'm wrong.